c_src/connect_q330.c

```diff
@@ -0,0 +1,87 @@
+
+#ifndef stdbool_h
```

Comment:
You should not need that, as all the headers provide their guards. Just:

#include <stdbool.h>
#include <stdio.h>
...

If that was in an example, whoever wrote that doesn't understand how this is handled with include guards.

Reply:
Fixed, thanks.